.github/workflows/scripts/register_buildpack/index.js

```diff
@@ -0,0 +1,244 @@
+const Path = require('path');
+const Schema = require('validate')
+const Toml = require('toml')
+
+const bodySchema = new Schema({
+    id: {
+        type: String,
+        required: true,
+    },
+    version: {
+        type: String,
+        match: /^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/,
```

Comment:
This is a gnarly version regex. Could you add a comment describing what it's ultimately validating? Maybe a set of tests around it.

Reply:
I can't take credit for this regex.  I stole it from the official semver website, because I don't trust myself 😄   I believe I have a test case that covers this.  I can add a comment with a reference/link to the semver documentation.